server_installation/topics/operator/command-options.adoc

```diff
@@ -18,5 +18,8 @@ A window opens where you can make changes.
 
 [NOTE]
 ====
-You can update the YAML file and changes appear in the {project_name} admin console, however changes to the admin console do not update the custom resource.
+You can update the Keycloak CR YAML file and changes will be applied to the deployment.
+
+However, updates to the other resources are not, currently, supported.
+Keycloak Realm, Client, User and Backup CRs only support basic creation and deletion without sync options.
```

Comment:
I'm afraid this is not accurate. Client and User support limited updatability. Backup is something else (it's not a KC resource) and is deprecated. We should mention only Realm CRs.

Reply:
fixed (hopefully) in case simply suggest a better wording and I'll incorporate it.